docker/oso-host-monitoring/src/start.sh

```diff
@@ -17,6 +17,10 @@ source /root/.bashrc
 # Configure the container
 time ansible-playbook /root/config.yml
 
+# Configure rkhunter
+echo '/usr/bin/ops-runner -f -t 60 -n monitoring.root.rkhunter.yml ansible-playbook /root/rkhunter.yml >> /var/log/rkhunter.yml.log &'
+/usr/bin/ops-runner -f -t 60 -n monitoring.root.rkhunter.yml ansible-playbook /root/rkhunter.yml >> /var/log/rkhunter.yml.log &
```

Comment:
We might not see much value from alerting based on this. I think the timeout on this should be increased or removed because just the ["Set initial values for rkhunter to use"](https://github.com/openshift/openshift-tools/pull/3302/files/#diff-251e4637cf03704fc835c6c60a7df056R72) and ["Run a no-log scan to get copies of passwd and group"](https://github.com/openshift/openshift-tools/pull/3302/files/#diff-251e4637cf03704fc835c6c60a7df056R74) tasks take around 2.5 minutes to complete on a fairly quiet node.

Reply:
The timeout is intended to bring visibility on a failed start. At the moment we have failed starts with no visibility for very long periods of time (one problem has been ongoing for the last 28 days).

I will update from 60s to 600s. If it takes longer than this then we can see which nodes are affected and extend accordingly.

It is more important that we have some monitoring quickly, than none like we have today.